numba/npyufunc/wrappers.py

```diff
@@ -337,6 +354,7 @@ def _build_wrapper(self, library, name):
 
         func.attributes.add("alwaysinline")
         wrapper = wrapper_module.add_function(fnty, name)
+        wrapper.linkage = 'weak_odr'
```

Comment:
A comment about the necessity of this would be useful.

Reply:
added comment in https://github.com/numba/numba/pull/4155/commits/37684eff506bdf7fbdaa00e373d06d9321d57409